ts/navigation/WalletNavigator.ts

```diff
@@ -50,6 +51,9 @@ const WalletNavigator = createStackNavigator(
     [ROUTES.PAYMENT_TRANSACTION_SUMMARY]: {
       screen: TransactionSummaryScreen
     },
+    [ROUTES.PAYMENT_TRANSACTION_ERROR]: {
+      screen: ErrorTransactionScreen
```

Comment:
To be consistent with TransactionSummaryScreen can't this be TransactionErrorScreen?